docs/docker-image-details.md

```diff
@@ -52,6 +55,10 @@ Changelog
 
 Changelog
 
+- trino:426-gravitino-0.3.1
+  - Fix some issues
+
```

Comment:
This need two blank lines. Otherwise,  they won't have a line when using markdown.